src/main/java/com/lina/airline/controller/AuthenticationController.java

```diff
@@ -0,0 +1,68 @@
+package com.lina.airline.controller;
+
+import com.lina.airline.dto.request.AuthenticationRequest;
+import com.lina.airline.dto.response.AuthenticationResponse;
+import com.lina.airline.utils.ApplicationConstantsUtils;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.crypto.SecretKey;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+
+public class AuthenticationController {
+    private final AuthenticationManager authenticationManager;
+    @Value(ApplicationConstantsUtils.JWT_SECRET_DEFAULT_VALUE)
+    private String jwtSecretValue;
+
+    @PostMapping("/login")
+    public ResponseEntity<AuthenticationResponse> login(@RequestBody AuthenticationRequest authRequest) {
+
+        String jwt = "";
+        Authentication authentication = UsernamePasswordAuthenticationToken.unauthenticated(authRequest.email(),
+                authRequest.password());
+        Authentication authenticationResponse = authenticationManager.authenticate(authentication);
+        if (authenticationResponse != null && authenticationResponse.isAuthenticated()) {
```

Comment:
Business Logic should not be added to the controller layer. Move it to the service layer. And also create a new Service for the JWT token generator.